### PR TITLE
MTM-44154 Document measurements and events while processing the uplink message for Sigfox and Loriot

### PR DESCRIPTION
**Pull request by ashhSAG93**
The goal of the PR is to document the measurements and events created on processing of the uplink message like position and signal strength for Sigfox and Loriot

JIRA tickets - MTM-44154, MTM-44155

### CODE REVIEW

File: content/protocol-integration/lora-loriot-bundle/loriot-overview.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The following sections describe how to:
 * [Configure Loriot agent endpoint credentials](#configure-loriot-credentials) in LORIOT Network Server.
 * [Assign the Loriot admin role](#assign-loriot-admin-role) to the user in {{< product-c8y-iot >}}.
 * [Register devices](#register-loriot-device) and visualize Loriot payload using {{< product-c8y-iot >}}.
+* [Standard measurements and events created while processing the uplink message](#uplink-message)
```
Comment (BeateRixen) [inline content/protocol-integration/lora-loriot-bundle/loriot-overview.md]: This does not fit the heading which says:
"The following sections describe how to:"
Not sure if this really needs to go here.

File: content/protocol-integration/sigfox-bundle/overview.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ The following sections describe how to:
 - [Create device protocols](#device-protocols) with {{< product-c8y-iot >}}'s device database.
 - [Register devices](#register-device-sigfox) and visualize the Sigfox payload using {{< product-c8y-iot >}}.
 - [Update devices](#old-registration) registered with the general device registration.
+- [Standard measurements and events on uplink message](#uplink-message)
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/overview.md:25]: See above

File: content/protocol-integration/lora-loriot-bundle/loriot-overview.md (hunk 1 above)
Comment (ashhSAG93) [reply content/protocol-integration/lora-loriot-bundle/loriot-overview.md]: Removed in the latest commit

File: content/protocol-integration/sigfox-bundle/overview.md (hunk 2 above)
Comment (ashhSAG93) [reply content/protocol-integration/sigfox-bundle/overview.md]: Removed in the latest commit

**Review by MWindel — APPROVED** [review]

**Review by miryalarahul — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**